.styleci.yml

```diff
@@ -6,5 +6,3 @@ finder:
     path:
         - "src"
 
-enabled:
-    - short_array_syntax
```

Comment:
not sure what happened, it looks like instead of rebase you added the commits from master as new commits in the branch. if you can fix it, cool, otherwise i can solve that before merging.

Reply:
Sorry, I had some personal stuff coming up last week and didn't get around checking this. Looks like my IDE messed up the rebase, I'm going to fix the issue. Thanks for your patience!

Reply:
no worries, whenver you have time

Reply:
@Radiergummi thank you so much for this change. Might you be in a position to rebase?
This would also really benefit our [SDK Generator](https://github.com/microsoft/kiota/)

Reply:
@dbu thinking of forking @Radiergummi's changes, rebasing and submitting a new PR. Thoughts? Or should we give this more time?

Reply:
glad if you want to do that. you can add the changes in a new commit and then the work of each contributor is still properly attributed.